README.md

```diff
@@ -59,83 +60,86 @@ should give output (among other transport plugins):
+
 ### Publisher (camera driver)
 
 Here is a list of the available encoding parameters:
 
-- ``encoding``: the libav (ffmpeg) encoder being used. The default is ``libx264``, which is on-CPU unaccelerated encoding.
+- ``encoder``: the libav (ffmpeg) encoder being used. The default is ``libx264``, which is on-CPU unaccelerated encoding.
   Depending on your hardware, your encoding options may include the hardware accelerated ``h264_nvenc`` or ``h264_vaapi``.
   You can list all available encoders with ``ffmpeg --codecs``. In the h264 row, look for ``(encoders)``.
-- ``preset``: default is empty (""). Valid values can be for instance ``slow``, ``ll`` (low latency) etc.
-   To find out what presets are available, run e.g.
-   ``ffmpeg -hide_banner -f lavfi -i nullsrc -c:v libx264 -preset help -f mp4 - 2>&1``
-- ``profile``: For instance ``baseline``, ``main``. See [the ffmpeg website](https://trac.ffmpeg.org/wiki/Encode/H.264).
-- ``tune``: See [the ffmpeg website](https://trac.ffmpeg.org/wiki/Encode/H.264). The default is empty("").
+- ``av_options``: default is empty (""). Comma-separeted list of valid
+   libav options in the form of (key:value),  e.g.:``'preset:ll,profile:main,crf:0'``.
+   See the ffmpeg documentation for [more](https://trac.ffmpeg.org/wiki/Encode/H.264).
 - ``gop_size``: The number of frames between keyframes. Default: 10.
    The larger this number the more latency you will have, but also the more efficient the compression becomes.
 - ``bit_rate``: The max bit rate [in bits/s] that the encoding will target. Default is ``8242880``.
-- ``crf``: Constant Rate Factor, affects the image quality. Value range is ``[0, 51]``; ``0`` is lossless, ``23`` is default, ``51`` is worst quality.
-- ``delay``: Not sure what it does, but doesn't help with delay. Default is empty ("").
-- ``pixel_format``: Forces a different pixel format for internal conversions. Experimental, don't use.
-- ``qmax``: Max quantization rate. Defaults to 10. See [ffmpeg documentation](https://www.ffmpeg.org/ffmpeg-codecs.html).
-   The larger this number, the worse the image looks, and the more efficient the encoding.
+- ``pixel_format``: Forces a different pixel for encoding.
+  This pixel format corresponds to the ``av_source_pixel_format`` in the
+  [ffmpeg\_encoder\_decoder documentation](https://github.com/ros-misc-utilities/ffmpeg_encoder_decoder).
+- ``qmax``: Max quantization rate. Defaults to 10.
+  See [ffmpeg documentation](https://www.ffmpeg.org/ffmpeg-codecs.html).
+  The larger this number, the worse the image looks, and the more efficient the encoding.
 - ``measure_performance``: For performance debugging (developers only). Defaults to false.
-- ``performance_interval``: How many frames to wait between logging performance data.
+- ``performance_interval``: How many frames to wait between logging performance data. Ignore.
```

Comment:
@berndpfrommer what do you mean by _ignore_?

Reply:
removed the parameter. It's developers only. I haven't switched on performance debugging for many years. It's probably completely broken.